src/components/preview/PreviewThumbnail.tsx

```diff
@@ -47,6 +49,7 @@ const categoryMap: Record<string, string> = {
 const PreviewThumbnail: React.FC<PreviewThumbnailProps> = ({ data }) => {
```

Comment:
FC 식으로 만드는건 지양하는 걸로 알고있슴다! 아래처럼 사용 부탁드려요!!

```suggestion
const PreviewThumbnail = ({ data }: PreviewThumbnailProps ) => {
```